### PR TITLE
An Introduction to WebAssembly - part 2

### PR DESCRIPTION
**Pull request by lucasm08**
This article is the sequel to my previous article on WebAssembly where I walk through the steps of building a react application the consumes a Wasm module. This article includes only one .png image which a browser screenshot of the resulting application.

### CODE REVIEW

File: articles/introduction-web-assembly-continued/index.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+
+`src/constants/index.js` :
+
+```javascript
+export const NUMBER_OF_SAMPLES = 1024;
+```
+
+This marks the end of our demo application, and if you followed all the steps above you should end up with something similar to [this](https://github.com/lucasm08/react-wasm-audio-rendering). Now, all that's left  is the start the app with `npm start`:
+
+![Web browser screenshot](browser-screenshot.png "")
```
Comment (hectorkambow) [inline articles/introduction-web-assembly-continued/index.md:487]: @lucasm08  - just wanted to check if there is an image missing from the article?

Comment (lucasm08) [reply articles/introduction-web-assembly-continued/index.md:487]: Nope! it's my mistake. I forgot to update the markdown after editing the image file name. oops!!

Comment (lucasm08) [reply articles/introduction-web-assembly-continued/index.md]: I fixed it. Everything should alright now

**Comment by hectorkambow** [comment]
Good afternoon @kurt - wanted to ask if you could give this article topic a tech review whenever you are free. 
Thank you 🚀

**Comment by lucasm08** [comment]
> This article needs a thorough review for grammar and usage. There are numerous sentences with missing or extra words. Go over it again, slowly and thoroughly, please.
> 
> I suggest you avoid introducing new information in the conclusion. The caveats you mention, such as a lack of tooling, the need for experimentation, etc., would be valuable in the introduction. The conclusion would be a good opportunity to reiterate the main themes and steps of the article.

**Comment by lucasm08** [comment]
Thanks for the feeback. I'm on it

**Comment by lucasm08** [comment]
@kurt-sectionio I'm done with the feedbacks

**Comment by lucasm08** [comment]
Thank you @kurt-sectionio for your patience. I have taken all of your feedback seriously I plan to not repeat those mistakes in my future articles. Sorry for the trivial grammar mistakes and typos. Since English is not my native language, I tend to make sometimes systematic translation from french to English which makes some of the sentences longer than they need to be. I'm strongly willing to improve my English skills to reduce as much as possible all of these mistakes. Thanks!

**Comment by lucasm08** [comment]
also, I made sure the demo application works well on my machine by following the exact same steps covered in this article.

**Pull request merged**